src/python.rs

```diff
@@ -234,6 +234,7 @@ impl<'p> Python<'p> {
     }
 
     /// Register `ffi::PyObject` pointer in release pool
+    #[allow(clippy::wrong_self_convention)]
```

Comment:
I'd like to eventually rename and and move those functions, but for now that's fine.